### ISSUE TITLE
WebPart Settings: not visible after adding to page with pnp-modern-search webpart 

### ISSUE THREAD

**Issue by Id10tSupport** [YannickRe/spfx-msgraph-peoplesearch#54]
**Describe the bug**
Same as bug reported in pnp-modern-search issues list

**To Reproduce**
This time, I created a new Team site and a new Communication site in my test tenant.
 -- looking to see if there was an issue with the site or the Site Pages library.
Created a new Site Page in both, then added the People Search webpart (after updating to the current version in the app catalog).
Published the page, refreshed, and then edited the page.  Could see the settings just fine.
Added the pnp-modern-search webpart, published the page and refreshed.
Went back into edit and now the setting for the people search are blank.

Occasionally get a message in the settings pane: 
"No settings available. The web part or dashboard card using this property pane was deleted."

**Expected behavior**
Settings should be visible for webpart when editing the page

**Screenshots**
![image](https://user-images.githubusercontent.com/102387183/161594270-68a42624-f8f6-4a7c-9493-7adc9ae6cf5b.png)

**Comment by YannickRe** [comment]
I was able to reproduce it. It happens because I was using react-ace 9.5.0 (the latest) and PnP Modern Search uses an older version, and this creates conflicts with both components loaded on the page.
It's also described here: https://github.com/microsoft-search/pnp-modern-search/issues/1964

As a temporary solution, I released v2.7.0 where I downgraded my web part to the same version as PnP Modern Search.
